Assets/Puppet/PuppetCrashes.cs

```diff
@@ -146,6 +148,11 @@ public void LastCrashReport()
     }
 
     public static ErrorAttachmentLog[] GetErrorAttachmentstHandler(ErrorReport errorReport)
+    {
+        return GetErrorAttachments();
+    }
+
+    private static ErrorAttachmentLog[] GetErrorAttachments()
     {
         return new ErrorAttachmentLog[]
         {
```

Comment:
Please read UI component before building attachments. It reads cached text and binary so test app doesn't send correct attachments when attachments are changed right before sending it.

Reply:
Fixed